src/Cassandra.IntegrationTests/Cassandra.IntegrationTests.csproj

```diff
@@ -9,9 +9,29 @@
     <PackageId>Cassandra.IntegrationTests</PackageId>
     <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
   </PropertyGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\Cassandra\Cassandra.csproj" />
-    <ProjectReference Include="..\Cassandra.Tests\Cassandra.Tests.csproj" />
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net452' ">
+    <ProjectReference Include="..\Cassandra.Tests\Cassandra.Tests.csproj">
+      <SetTargetFramework>TargetFramework=net452</SetTargetFramework>
+    </ProjectReference>
+    <ProjectReference Include="..\Cassandra\Cassandra.csproj">
+      <SetTargetFramework>TargetFramework=net45</SetTargetFramework>
+    </ProjectReference>
+  </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp2.1' ">
+    <ProjectReference Include="..\Cassandra.Tests\Cassandra.Tests.csproj">
+      <SetTargetFramework>TargetFramework=netcoreapp2.1</SetTargetFramework>
+    </ProjectReference>
+    <ProjectReference Include="..\Cassandra\Cassandra.csproj">
+      <SetTargetFramework>TargetFramework=netstandard2.0</SetTargetFramework>
```

Comment:
Nice trick to test multiple targets

Reply:
It would be nice to have a way to test that `netcoreapp2.0` on tests map to `netstandard1.5` in case we inadvertently change something in the future that causes both test targets to test `netstandard2.0`... but its not something we have to do now.

Reply:
I've added a `TargetTests` class that does this if I understood correctly. Can you check if that's what you were thinking of?

Reply:
Yes! 👍